src/test/java/com/epam/integration/tests/IntegrationTest.java

```diff
@@ -1,5 +1,55 @@
 package com.epam.integration.tests;
 
-public interface IntegrationTest {
+import com.epam.data.RoadAccident;
+import com.epam.dataservice.PoliceForceService;
+import com.epam.report.DataReportGenerator;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
+@RunWith(MockitoJUnitRunner.class)
+public class IntegrationTest {
+
+    private DataReportGenerator dataReportGenerator;
+
+    private static List<String> inputFileNames;
+
+    private final String testInputFileName = "src/main/resources/DfTRoadSafety_Accidents_2009.csv";
+    private final String testOutputMorningFileName = "testOut1.csv";
+    private final String testOutputEveningFileName = "testOut2.csv";
+
+
+    @Before
+    public void setUp() throws Exception {
+        inputFileNames = new ArrayList<>();
+        inputFileNames.add(testInputFileName);
+
+        dataReportGenerator = new DataReportGenerator(inputFileNames, testOutputMorningFileName, testOutputEveningFileName);
+    }
+
+    @Test
+    public void should_find_by_index7(){
+        try {
+            dataReportGenerator.generateReport();
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+
+        File f1 = new File(testOutputMorningFileName);
+        File f2 = new File(testOutputEveningFileName);
+
+        Assert.assertTrue(f1.exists() && !f1.isDirectory());

```

Comment:
I generally try to avoid using assertTrue for assertion. This will give message like "Something is false when expected true" - not so much use when you see it in build log. To understand why it's failed you will need to run it locally. And as it's integration test - you might get different results locally... So you will spend long time trying to understand what is wrong (believe me - I have been there) 

Here you are asserting boolean so it's fine, but I would add message at least to make debug easier.
And split it on two parts
assertTrue("Does not exist", f1.exists());
assertFalse("Is directory" , f1.isDirectory()); 

What's actually gives you perfect messages without additional effort is using matchers. As example assetThat(something, is(somethingElse)); 

Whereever it's possible I'm trying to use them